src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -272,6 +265,37 @@ void getMultipleHearingLocations_shouldReturnCorrespondingMultipleEpimsIdForVenu
 
     }
 
+    @DisplayName("Regional hearing location Test")
+    @Test
+    void getRegionalHearingLocations_shouldReturnCorrespondingEpimsIdsForVenuesWithSameRpc() {
+        final SscsCaseData caseData = SscsCaseData.builder()
+            .dwpIsOfficerAttending("No")
+            .regionalProcessingCenter(RegionalProcessingCenter.builder().name("SSCS Leeds").build())
```

Comment:
Minor, can you put anything after builder() on a new line for readability and to make adding fields in the future easier